code/modules/mob/living/simple_animal/friendly/mouse.dm

```diff
@@ -45,6 +45,7 @@
 	var/is_fat = 0
 	var/can_chew_wires = 0
 	var/splat = 0
+	var/infectable = 0
```

Comment:
it would appear you've changed the infectability default from true to false?

also where's the actual toggle button for administration? Unless you are meaning "just VV edit every mouse"

Reply:
That's the intention, and yes VV the mouse you want to be infectable